src/assets/styles/base.scss

```diff
@@ -276,3 +276,12 @@ ul.list {
 .vue-affix {
   width: 282px;
 }
+
+
+.v-size--x-small {
```

Comment:
This is fine, but please add a comment indicating which element this is targeting and why we are overriding styles.

Reply:
Perfect, thanks!